linux-cachyos/PKGBUILD

```diff
@@ -141,9 +141,27 @@ _build_nvidia_open=${_build_nvidia_open-}
 # Build a debug package with non-stripped vmlinux
 _build_debug=${_build_debug-}
 
+# Enable AUTOFDO_CLANG for the first compilation to create a kernel, which can be used for profiling
+# This only works for Clang compiled kernels, therefore thin or full lto needs to be set in _use_llvm_lto
+# Workflow:
+# https://cachyos.org/blog/2411-kernel-autofdo/
+# 1. Compile Kernel with _autofdo=y and _build_debug
+# 2. Boot the kernel in QEMU or on your system, see Workload
+# 3. Profile the kernel and convert the profile, see Generating the Profile for AutoFDO
+# 4. Put the profile into the sourcedir and name the profile "perf.afdo" (You can also change the name in _autofdo_profile_name)
+# 5. Enable _autofdo_use
+_autofdo=${_autofdo-}
+
+# Enable after you have put the profile into the sourcedir
+_autofdo_use=${_autofdo_use-}
```

Comment:
Can't we just check availability of a profile file?

Reply:
I mean, if that works - sure. Feel free to push it, if its working properly.

Reply:
Seems to be working, but I haven't tested a full rebuild.